src/plugins/empty-org-recycle-bin/index.ts

```diff
@@ -0,0 +1,42 @@
+import { BrowserforcePlugin } from '../../plugin';
+
+const DELETE_EVENT_OBJECT = 'DeleteEvent';
+const LIST_VIEW_ALL_DELETE_EVENTS = 'AllDeleteEvents';
+const DELETE_EVENT_EMPTY_ALL_ACTION = 'DeleteEventEmptyAllAction';
+
+const PATHS = {
+  BASE: 'lightning/o/DeleteEvent/home'
+};
+
+const SELECTORS = {
+  EMPTY_BUTTON: '.modal-container .modal-footer button.uiButton:last-of-type'
+};
+
+export class EmptyOrgRecycleBin extends BrowserforcePlugin {
+  public async retrieve(definition?: Config): Promise<Config> {
+    return { enabled: false };
+  }
+
+  public async apply(config: Config): Promise<void> {
+    const deleteEventEmptyAllButtonSelector = `a.forceActionLink[title="${await this.determineDeleteEventEmptyAllActionLabel()}"]`;
```

Comment:
Given that we assume the "Empty" button in the dialog is the last button,
can we maybe also just assume the "Empty Org Recycle Bin" is the last item of the `forceActionsContainer`?
As far as I can see, the button is there regardless if there are items in the Recycle Bin or not and I'm able to click it even if the Recycle Bin is empty.

```suggestion
    const deleteEventEmptyAllButtonSelector = `ul.forceActionsContainer li:last-of-type a`;
```

In that case we could go without using the UI API for simplicity and performance.
But it's great to see a use case of this API. 👏